### ISSUE TITLE
Feature request: support multi page diffing of PDF with multiple pages

### ISSUE THREAD

**Issue by ankush** [HyGear/diff-dwg#5]
I can work on this and send a PR if you want to implement it.

**Comment by HyGear** [comment]
I tried doing a multi-page diff in the past but never got it to work correctly. If a new page is added to the document it will shift the remaining pages and the comparison will not work if you are doing a page to page comparison.

The only way I could think of to get around this without a huge CPU load was to compare text between PDF files. The idea was to compare the text and determine which pages match between the two PDF files then use the anaglyph algorithm to make the composite comparison image. Unfortunately I discovered that matching text between documents is also difficult so I just focused on making it work for single documents.
